app/controllers/course/statistics_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+class Course::StatisticsController < Course::ComponentController
+  before_action :authorize_read_statistics!
+
+  def student
+    preload_levels
+    course_users = current_course.course_users.with_approved_state.includes(:groups)
+    @students = course_users.students.without_phantom_users.ordered_by_experience_points
+    @phantom_students = course_users.phantom.students.ordered_by_experience_points
```

Comment:
Actually no need for a `.phantom` here, you can use `@phantom_students = course_users - @students`